web/source/kmwembedded.ts

```diff
@@ -250,7 +250,11 @@ namespace com.keyman.text {
    * correctOSKTextSize handles rotation event -- currently rebuilds keyboard and adjusts font sizes
    */
   keymanweb['correctOSKTextSize']=function() {
-    if(osk && osk.vkbd && osk.vkbd.adjustHeights(osk)) {
+    let osk = keymanweb.osk;
+    if(osk && osk.vkbd && osk.vkbd.adjustHeights(osk.getKeyboardHeight())) {
+      var b: HTMLElement = osk._Box, bs=b.style;
+      bs.height=bs.maxHeight=osk.vkbd.computedAdjustedOskHeight(this.getHeight())+'px';
```

Comment:
The new lines of the inner block are relocated from within `VisualKeyboard.adjustHeights`, removing the need for that function to reference this class's `_Box` field.  (It's _very_ inconvenient for a few specific calls elsewhere.)